src/app/donation-start/donation-start.component.ts

```diff
@@ -954,21 +954,19 @@ export class DonationStartComponent implements AfterContentChecked, AfterContent
     }
   }
 
-  private loadAuthedPersonInfo(id: string, jwt: string) {
-    this.identityService.get(id, jwt).subscribe((person: Person) => {
-      this.personId = person.id; // Should mean donations are attached to the Stripe Customer.
-      this.personIsLoginReady = true;
-
-      if (environment.creditDonationsEnabled && person.cash_balance && person.cash_balance[this.campaign.currencyCode.toLowerCase()]! > 0) {
-        this.creditPenceToUse = parseInt(
-          person.cash_balance[this.campaign.currencyCode.toLowerCase()]!.toString() as string,
-          10
-        );
-        this.stripePaymentMethodReady = true;
-        this.setConditionalValidators();
-      }
+  loadAuthedPersonInfo(id: string, jwt: string) {
+    if(this.identityService) {
+      this.identityService.get(id, jwt).subscribe((person: Person) => {
+        this.personId = person.id; // Should mean donations are attached to the Stripe Customer.
+        this.personIsLoginReady = true;
+        this.prefillRarelyChangingFormValuesFromPerson(person);
+        this.loadFirstSavedStripeCardIfAny(id, jwt);
+      });
+    }
+  }
 
-      // Pre-fill rarely-changing form values from the Person.
+    // Pre-fill rarely-changing form values from the Person.
+    prefillRarelyChangingFormValuesFromPerson(person: Person) {
```

Comment:
Turning comments into function names is good, the comment is now redundant. 
```suggestion
    prefillRarelyChangingFormValuesFromPerson(person: Person) {
```